bootstrap/kubeadm/types/upstreamv1beta2/types.go

```diff
@@ -56,6 +56,11 @@ type InitConfiguration struct {
 	// a secret in the cluster during the uploadcerts init phase.
 	// +optional
 	CertificateKey string `json:"certificateKey,omitempty"`
+
+	// Patches contains options related to applying patches to components deployed by kubeadm during
+	// "kubeadm join". The minimum kubernetes version needed to support Patches is v1.22
+	// +optional
+	Patches *Patches `json:"patches,omitempty"`
```

Comment:
`InitConfiguration.Patches` and `JoinConfiguration.Patches` were only added to the kubeadm v1beta3 types. The v1beta2 types do not contain these fields.

https://github.com/kubernetes/kubernetes/blob/v1.23.1/cmd/kubeadm/app/apis/kubeadm/v1beta2/types.go